tests/test_fields.py

```diff
@@ -224,27 +257,42 @@ def test_static_rules_bad(value, func, assertion):
     '8k;abc;abc'
 ])
 def test_ignore_cell(cell):
-    assert not field.Ignore().check_cell(cell)
+    fld = field.Ignore()
+    assert not fld.check_cell(cell)
+    file_cell_harness(cell, fld)
 
 
 def test_column_compare_rule_good():
+    value = ('1', {'x': '0'})
     x = field.Field(rules=[rules.cell.GreaterThanColumn('x')])
-    assert not x.check_cell(('1', {'x': '0'}))
+    assert not x.check_cell(value)
+
+
+def test_column_compare_rule_good_files():
+    value = ['1', '0']
+    lay = {'c1': field.Field(rules=[rules.cell.GreaterThanColumn('c2')]), 'c2': field.Integer(1)}
+    file_cell_harness(value, lay)
```

Comment:
```suggestion
    file_row_harness(value, lay)
```

We should try this to test if we can revert the cell function back to taking only single values.

Reply:
Swapped to `file_row_harness` in this and similar tests, works great!